tangos/tools/ahf_merger_tree_importer.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+import tangos as db
+import os
+from ..input_handlers import ahf_trees as at
+from ..log import logger
+from ..core import get_or_create_dictionary_item
+from ..core.halo_data import HaloLink, HaloProperty
+from .. import config
+from . import GenericTangosTool
+from six.moves import xrange
+import re
+
+class AHFTreeImporter(GenericTangosTool):
+    tool_name = 'import-ahf-trees'
+    tool_description = 'Import properties that were calculated by the halo finder'
```

Comment:
Is this the correct help text? Presumably it's importing the merger tree in fact.

Reply:
ups, indeed, I will change that text